lib/platform-stats.js

```diff
@@ -199,3 +316,119 @@ export const updateTopMeasurementParticipants = async (createPgClient) => {
     Sentry.captureException(err)
   }
 }
+
+/**
+ * @param {import('pg').Client} pgClient
+ */
+export const aggregateAndCleanupRecentData = async (pgClient) => {
+  try {
+    await pgClient.query('BEGIN')
+
+    // The ON CONFLICT theoretically should not happen, because it's two days old data,
+    // which should not be updated anymore. But it's a good practice to have it.
+    await pgClient.query(`
+      WITH rsdn_summary AS (
```

Comment:
What does `rsdn` stand for?

Reply:
Ah I think this was a remnant of my local tests for several different schemas, will fix

Reply:
ping

Reply:
I think this was already addressed (switched ot rsd) before your last comment?

But I also just pushed https://github.com/filecoin-station/spark-evaluate/pull/311/commits/e3845c9bd36cfddb719459fe72c3f2389b94ecda based on a below comment, and spelled out the table name in the "_summary", to align with your suggestion for it for the other summary variable

Reply:
what does rsd stand for?